zerver/views/email_mirror.py

```diff
@@ -13,11 +13,10 @@
 
 @internal_notify_view(False)
 @has_request_variables
-def email_mirror_message(request, data=REQ(validator=check_dict([
-        ('recipient', check_string),
-        ('msg_text', check_string),
-]))):
-    # type: (HttpRequest, Dict[str, str]) -> HttpResponse
+def email_mirror_message(request: HttpRequest,
+                         data: Dict[str, str]=REQ(validator=check_dict([
+                             ('recipient', check_string),
+                             ('msg_text', check_string), ]))) -> HttpResponse:
```

Comment:
Extra white space before closing ].

Reply:
@Kiy4h Regarding the PEP-8 thing, you can remove the comma and the space and the linter shouldn't complain.